src/_base/harness/attributes/docker-base.yml

```diff
@@ -167,6 +175,12 @@ attributes:
           enabled: false
         mysql:
           enabled: false
+        php-base:
+          environment:
+            TIDEWAYS_ENABLED: "= @('php.ext-tideways.enable') ? 'true' : 'false'"
+        console:
+          environment:
+            TIDEWAYS_ENABLED: "= @('php.ext-tideways.cli.enable') ? 'true' : 'false'"
```

Comment:
ditto with the other comment but perhaps with the cli.enabled twig stuff still

Reply:
Updated :+1: